lib/parse.js

```diff
@@ -2109,7 +2109,7 @@ function parse($TEXT, options) {
         return function_(AST_Accessor, is_generator, is_async);
     });
 
-    var object_or_object_destructuring_ = embed_tokens(function() {
+    var object_or_destructuring_ = embed_tokens(function object_or_destructuring_() {
```

Comment:
As for `embed_tokens()`, it sets the positional information for the parsed expression and saves a bit of boilerplate code.